cves/kernel/CVE-2016-9754.yml

```diff
@@ -149,10 +151,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 'The issue was reported by a Red Hat employee'
+  automated: false
+  contest: false
+  developer: false
```

Comment:
Was the method of discovery mentioned by the employee?

Reply:
It was not. I assume it was a manual find.